calicoctl/calicoctl.go

```diff
@@ -48,7 +49,7 @@ Options:
 	if logLevel := arguments["--log-level"]; logLevel != nil {
 		parsedLogLevel, err := log.ParseLevel(logLevel.(string))
 		if err != nil {
-			fmt.Printf("Unknown log level: %s, expected one of: \n" +
+			fmt.Printf("Unknown log level: %s, expected one of: \n"+
```

Comment:
Interesting... I didn't realize that was standard Go formatting.

Looks pretty weird to me, but so be it :(